meta-resin-common/recipes-containers/docker/docker/0001-bucket-correct-broken-unaligned-load-store-in-armv5.patch

```diff
@@ -0,0 +1,148 @@
+From 494b0c6feb74a8a4783bc2cc6f0ff406dbec9973 Mon Sep 17 00:00:00 2001
+From: Lorenzo Stoakes <lstoakes@gmail.com>
+Date: Mon, 1 Aug 2016 13:40:27 +0100
+Subject: [PATCH] bucket: correct broken unaligned load/store in armv5
+
+armv5 devices and older (i.e. <= arm9 generation) require addresses that are
+stored to and loaded from to to be 4-byte aligned.
+
+If this is not the case the lower 2 bits of the address are cleared and the load
+is performed in an unexpected order, including up to 3 bytes of data located
+prior to the address.
+
+Inlined buckets are stored after their key in a page and since there is no
+guarantee that the key will be of a length that is a multiple of 4, it is
+possible for unaligned load/stores to occur when they are cast back to bucket
+and page pointer types.
+
+The fix adds a new field to track whether the current architecture exhibits this
+issue, sets it on module load for ARM architectures, and then on bucket open, if
+this field is set and the address is unaligned, a byte-by-byte copy of the
+inlined bucket is performed.
+
+Ref: http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.faqs/ka15414.html
```

Comment:
it misses an Upstream-status field as documented here: http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines